src/goalServices/getGoalsMissingDataForActivityReportSubmission.test.js

```diff
@@ -79,37 +79,43 @@ describe('getGoalsMissingDataForActivityReportSubmission', () => {
       where: {
         goalId: goalOne.id,
       },
+      individualHooks: true,
```

Comment:
Is there any worry that these hooks could cause unforeseen issues? Almost wondering if we want to wrap in try catch with the catch failing the test.